ui/pages/confirm-add-suggested-nft/index.scss

```diff
@@ -0,0 +1,124 @@
+.confirm-add-suggested-nft {
+  margin: 16px;
+  box-shadow: 0 4px 16px rgba(0, 0, 0, 0.15);
+
+  &__header {
+    padding-bottom: 8px;
+    border-bottom: 1px solid var(--color-border-muted);
+  }
+
+  &__nft-list {
+    padding: 8px;
+    display: flex;
+    flex-flow: column nowrap;
+    width: 100%;
+  }
+
+  &__nft-list-item {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__nft-details {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+
+    &--error {
+      opacity: 0.5;
+    }
+
+    margin: 0;
+  }
+
+  &__nft-data {
+    display: flex;
+    align-items: center;
+    padding: 8px;
+  }
+
+  &__nft-description,
+  &__nft-standard {
+    min-width: 0;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  &__nft-image {
+    margin-right: 12px;
+    width: 48px;
+    border-radius: 8px;
+    flex: 0 0 auto;
+  }
```

Comment:
Pretty sure a chunk of these styles can be removed by using <Box> and specifying img as the tag.

Reply:
Not sure I follow this one?

Reply:
Second guessing myself here anyways. resolving